src/lib/airtable/patches/README.md

```diff
@@ -3,7 +3,7 @@ This folder contains patches that should be run after regenerating the airtable
 
 ## Directions to apply patch
 - Navigate to the root directory `meepanyar/`.
-- Apply patch in the `src/lib/airtable/patches/` directory by running the following command: `git apply --reject --whitespace=fix src/lib/airtable/patches/*.patch`
+- Apply patch in the `src/lib/airtable/patches/` directory by running the following command in the root directory `meepanyar/`: `npm run apply-schema-patches`.
```

Comment:
huge nit: add a link somewhere to explain what patches are for the curious

Reply:
Link to patch docs is at the bottom of "Add Additional Patches"